docs/configuration/options.md

```diff
@@ -123,7 +129,7 @@ By default JSON Web Tokens are encrypted (JWE). We recommend you keep this behav
 
 ```js
 jwt: {
-  // A secret to use for key generation. Defaults to the top-level `secret`.
+  // A secret to use for JWT encryption. Use `NEXTAUTH_SECRET` environment variable instead.
```

Comment:
What does this mean? Shouldn't this just use `NEXTAUTH_SECERT` here then like it says?

Reply:
It's a legacy option and is not necessary anymore, as we derive all keys from the secret. I think to push the "happy path", I'll just remove this option. :+1: